plugins/services/src/js/stores/MarathonStore.js

```diff
@@ -495,6 +501,12 @@ class MarathonStore extends GetSetBaseStore {
         };
       }
 
+      if (item instanceof Application) {
+        map[item.getName().toLowerCase()] = {
+          images: item.getImages()
```

Comment:
Framework `if` above also has `health` and `snapshot` keys. adding these here fixes the broken test.

Reply:
Are we sure Applications have health and snapshots?

Reply:
I think the correct question should be "why is there this `if framework` clause in the first place? why do we need it?" can you figure that out please? Also `getFrameworkName` returns the predefined framework name, not the services ID… so shouldnt that block be broken if thee are multiple frameworks of the same kind installed? 🤔

Reply:
About the first question: I think we didn't want non-frameworks to have images.

Reply:
About the second question: it is actually the service id.
```
  processMarathonGroups(data) {
    const groups = new ServiceTree(data);
    const apps = groups.reduceItems(function(map, item) {
      if (item instanceof Framework) {
        console.log(item.getFrameworkName());
        map[item.getFrameworkName().toLowerCase()] = {
          health: item.getHealth(),
          images: item.getImages(),
          snapshot: item.get()
        };
      }
...
```
![Screenshot from 2019-04-23 13-09-42](https://user-images.githubusercontent.com/40791275/56573115-13692300-65c9-11e9-804f-625c2b613fe6.png)